lib/galaxy/web_stack/__init__.py

```diff
@@ -220,372 +185,12 @@ def set_postfork_server_name(self, app):
-                        }
-                    )
-                    server_names.append(self.server_name_template.format(**facts))
-                self._farm_server_names_dict[farm_name] = server_names
-        return self._farm_server_names_dict
-
-    @property
-    def instance_id(self):
-        if not self._is_mule:
-            instance_id = uwsgi.worker_id()
-        elif self._farm_name:
-            return self._mule_index_in_farm(self._farm_name) + 1
-        else:
-            instance_id = uwsgi.mule_id()
-        return instance_id
-
-    def log_startup(self):
-        msg = [f"Galaxy server instance '{self.config.server_name}' is running"]
-        # Log the next messages when the first worker finishes starting. This
-        # may not be the first to finish (so Galaxy could be serving already),
-        # but it's a good approximation and gives the correct root_pid below
-        # when there is no master process.
-        if not self._is_mule and self.instance_id == 1:
-            # We use the same text printed by Paste to not break scripts
-            # grepping for this line. Here root_pid is the same that gets
-            # written to file when using the --pidfile option of uwsgi
-            root_pid = uwsgi.masterpid() or os.getpid()
-            msg.append("Starting server in PID %d." % root_pid)
-            for s in UWSGIApplicationStack._serving_on():
-                msg.append(f"serving on {s}")
-            if len(msg) == 1:
-                msg.append("serving on unknown URL")
-        log.info("\n".join(msg))
-
-    def start(self):
-        # Does a generalized `is_worker` attribute make sense? Hard to say w/o other stack paradigms.
-        if self._is_mule and self._farm_name:
-            # used by main.py to send a shutdown message on termination
-            os.environ["_GALAXY_UWSGI_FARM_NAMES"] = ",".join(self._farms)
-        super().start()
-
-    def in_pool(self, pool_name):
-        if not self._is_mule:
-            return False
-        else:
-            return pool_name in self._farms
-
-    def pool_members(self, pool_name):
-        return self._farm_server_names.get(pool_name, None)
-
-    def workers(self):
-        return uwsgi.workers()
-
-    @property
-    def facts(self):
-        facts = super().facts
-        if not self._is_mule:
-            facts.update(
-                {
-                    "pool_name": "web",
-                    "server_id": uwsgi.worker_id(),
-                }
-            )
-        else:
-            facts.update(
-                {
-                    "pool_name": self._farm_name,
-                    "server_id": uwsgi.mule_id(),
-                }
-            )
-        facts["instance_id"] = self.instance_id
-        return facts
-
-    def shutdown(self):
-        super().shutdown()
-
-
-class PasteApplicationStack(ApplicationStack):
-    name = "Python Paste"
+class WebApplicationStack(ApplicationStack):
```

Comment:
Unused?

Reply:
Should be used by non-gunicorn asgi application servers (running my dev instance with uvicorn for instance), but I'll double check